client/app/pages/alert/components/Query.jsx

```diff
@@ -27,9 +28,9 @@ export default function QueryFormItem({ query, queryResult, onChange, editMode }
       <small>
         <WarningFilledIcon className="warning-icon-danger" /> This query has no <i>refresh schedule</i>.{" "}
         <Tooltip title="A query schedule is not necessary but is highly recommended for alerts. An Alert without a query schedule will only send notifications if a user in your organization manually executes this query.">
-          <a>
+          <PlainButton type="link">
```

Comment:
Is it correct to use a link `PlainButton` here? Don't they have a `role` of `button` associated when them?

Reply:
This `type` in `<PlainButton>` is only useful for styling the "text button" as an "Ant design link-looking text button". In this context it felt appropriate, as this is the standard followed in other parts of the codebase, e.g. `EmptyState`